frontend/src/components/Post/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from 'react';
+import Media from 'react-bootstrap/Media';
+import styles from './styles.module.sass';
+import { IUser } from '../../common/models/user/IUser';
+import { getAmPmTimeFromDate } from '../../common/helpers/dateHelper';
+
+interface IProps {
+  user: Partial<IUser>;
+  text: string;
+  sendedAt: Date;
+}
+
+const Post: FunctionComponent<IProps> = ({ user, text, sendedAt }) => (
```

Comment:
we should use the component as <Post post={...}>
now it's <Post user={...} text={...} sendedAt={...}>